subsys/bluetooth/controller/hal/nrf5/radio.c

```diff
@@ -680,19 +680,13 @@ u32_t radio_tmr_start(u8_t trx, u32_t ticks_start, u32_t remainder)
 	NRF_PPI->CH[1].TEP = (u32_t)&(NRF_TIMER0->TASKS_START);
 	NRF_PPI->CHENSET = PPI_CHEN_CH1_Msk;
 
+	NRF_PPI->CH[0].EEP = (u32_t)&(NRF_TIMER0->EVENTS_COMPARE[0]);
 	if (trx) {
```

Comment:
break a line before the if. (the condition is not applied on previous assignment)